docs/resources/security_monitoring_rule.md

```diff
@@ -2,12 +2,12 @@
 page_title: "datadog_security_monitoring_rule Resource - terraform-provider-datadog"
 subcategory: ""
 description: |-
-  Provides a Datadog Security Monitoring Rule API resource. This can be used to create and manage Datadog security monitoring rules. To change settings for a default rule use datadogsecuritydefault_rule instead.
+  Provides a Datadog Security Monitoring Rule API resource. This can be used to create and manage Datadog security monitoring rules. To change settings for a default rule use datadog_security_default_rule instead.
```

Comment:
WTF is there some markdown rendering already happening in here ??

Reply:
... yeah I noticed that was weird. It was like it was trying to fix the word, until I added the backticks to the description

Reply:
Yeah and now the backticks are removed...